### DISCUSSION TITLE
Mise à jour d'un formulaire

### DISCUSSION THREAD

**Discussion by maxpouliot** [MTESSDev/FRW#5] [Annonces]
Dans le cadre d'une MAJ importante d'un formulaire, ou par exemple on ajoute ou un supprime un champ à un formulaire. 
Il arrive quoi avec les formulaires commencés mais non terminés? 

Merci!

**Comment by anisite** [comment]
Duplicate https://github.com/MTESSDev/FRW/discussions/3
